Modules/_blake2/blake2s_impl.c

```diff
@@ -159,6 +159,10 @@ py_blake2s_new_impl(PyTypeObject *type, PyObject *data, int digest_size,
     if (leaf_size_obj != NULL) {
         leaf_size = PyLong_AsUnsignedLong(leaf_size_obj);
         if (leaf_size == (unsigned long) -1 && PyErr_Occurred()) {
+            if (PyErr_ExceptionMatches(PyExc_OverflowError)) {
+                PyErr_SetString(PyExc_OverflowError,
+                                "leaf_size must be between 0 and 2**32-1");
```

Comment:
so do you suggest to remove the changes in `py_blake2b_new_impl` and `py_blake2s_new_impl`, and patch these functions as part of the issue of replacing OverflowError with ValueError for negative values?